src/molecules/formfields/CheckBoxField/index.js

```diff
@@ -13,17 +13,18 @@ function CheckboxField( props ) {
 
   return (
     <label
-      className={styles.checkbox}
+      className={styles['checkbox']}
       htmlFor={`checkbox-${input.name}`}
     >
       <input
         type='checkbox'
         id={`checkbox-${input.name}`}
+        className={styles['checkbox-input']}
         checked={input.value}
         {...input}
       />
       <span
-        className={classnames(styles.checkboxLabel, styles[fontStyle])}
+        className={classnames(styles['checkbox-label'], styles[fontStyle])}
```

Comment:
I think the intent is to be able to pass in different fontStyles in per checkbox? not sure.